src/openms/include/OpenMS/ANALYSIS/ID/SiriusAdapterAlgorithm.h

```diff
@@ -53,6 +56,38 @@ namespace OpenMS
       /// default constructor
       SiriusAdapterAlgorithm();
 
+      /*
+       * Accessors for Preprocessing Parameters
+       */
+      bool isFeatureOnly() const { return preprocessing.getValue("feature_only").toBool(); }
+      UInt getFilterByNumMassTraces() const { return preprocessing.getValue("filter_by_num_masstraces"); }
+      double getPrecursorMzTolerance() const { return preprocessing.getValue("precursor_mz_tolerance"); }
+      double getPrecursorRtTolerance() const { return preprocessing.getValue("precursor_rt_tolerance"); }
+      bool precursorMzToleranceUnitIsPPM() const { return preprocessing.getValue("precursor_mz_tolerance_unit") == "ppm"; }
+      bool isNoMasstraceInfoIsotopePattern() const { return preprocessing.getValue("no_masstrace_info_isotope_pattern").toBool(); };
+      int getIsotopePatternIterations() const { return  preprocessing.getValue("isotope_pattern_iterations"); }
+
+      /*
+       * Accessors for Sirius Parameters
+       */
+      int getNumberOfCandidates() const { return nightsky_sirius.getValue("candidates");  }
+
+      /**
+       * Updates all parameters that already exist in this DefaultParamHandler
+       * with the values provided by the input param object.
+       *
+       * @param param The Param object supplying updated parameter values. Keys that exist in the param
+       *        parameter, but not in this DefaultParamHander, are ignored.
+       */
+      void updateExistingParameter(const Param &param);
+
+      /**
+       * Checks whether this DefaultParamHandler has a ParamEntry with the provided name.
+       * @param name The name of the ParamEntry that should be checked for its existence in this DefaultParamHandler
+       * @return Whether this DefaultParamHandler has an ParamEntry for the provided name.
+       */
+      bool hasFullNameParameter(const String &name) const;
```

Comment:
can you check which of these should be exposed to Python? And then add these to the `.pxd` ?